sqla_yaml_fixtures.py

```diff
@@ -1,13 +1,19 @@
 __version__ = (0, 1, 2)
 
-import functools
+try:
+    # Python 3
+    from functools import lru_cache
+except ImportError:
+    # For Python 2: 
+    # use https://pypi.python.org/pypi/backports.functools_lru_cache
+    # pip install backports.functools_lru_cache
+    from backports.functools_lru_cache import lru_cache
 
 import yaml
 import sqlalchemy
 from sqlalchemy.orm.relationships import RelationshipProperty
 
 
-
```

Comment:
hmm, thought you were PEP8-thorough, so just made changes so we get no flake8 warnings....just clean Python, I also saw the warnings in my IDE (IntelliJ). But no need for `flake8`, it can be very annoying as well, as the CI may fail on a single space on a blank line...